aggregator/generic_elem.go

```diff
@@ -470,7 +489,12 @@ func (e *GenericElem) processValueWithAggregationLock(
 			continue
 		}
 		if !e.parsedPipeline.HasRollup {
-			flushLocalFn(fullPrefix, e.id, e.TypeStringFor(e.aggTypesOpts, aggType), timeNanos, value, e.sp)
+			switch e.idMutationType {
+			case IDMutationDisabled:
```

Comment:
Seems more clear to call the enum `IDPrefixSuffixType` and have the values be `NoPrefixNoSuffix` and `WithPrefixWithSuffix`.